x/spec/types/spec.go

```diff
@@ -36,6 +40,18 @@ func (spec Spec) ValidateSpec(maxCU uint64) (map[string]string, error) {
 	if spec.ReliabilityThreshold == 0 {
 		return details, fmt.Errorf("ReliabilityThreshold can't be zero")
 	}
+	if len(spec.Contributor) > 0 {
+		for _, contributorAddr := range spec.Contributor {
+			_, err := sdk.AccAddressFromBech32(contributorAddr)
+			if err != nil {
+				return details, fmt.Errorf("spec contributor is not a valid account address %s in list: %s", contributorAddr, strings.Join(spec.Contributor, ","))
+			}
+		}
+	}
+
+	if spec.ContributorPercentage > 0.8 || (spec.ContributorPercentage < 1.0/ContributorPrecision && spec.ContributorPercentage != 0) {
```

Comment:
(spec.ContributorPercentage < 1.0/ContributorPrecision && spec.ContributorPercentage != 0) -> 
spec.ContributorPercentage > 1.0/ContributorPrecision

Reply:
this is not the same condition... greater than 1.0/contribution is fine, the problem is less than that is not 0